README.md

```diff
@@ -5,7 +5,7 @@
 This repository contains HASH's open-source code, documentation, and other key resources, including:
```

Comment:
@Alfred-Mountfield If you want to add 'open-source and public' here (following suggestion by Andre in Slack discussion just now) that's fine.

Reply:
```suggestion
This repository contains HASH's open-source and public code, documentation, and other key resources, including:
```